google/cloud/bigtable/table.py

```diff
@@ -1078,9 +1081,15 @@ def _do_mutate_retryable_rows(self):
                 client_info=data_client._client_info,
             )
 
-        responses = data_client._inner_api_calls["mutate_rows"](
-            mutate_rows_request, retry=None
-        )
+        try:
+            responses = data_client._inner_api_calls["mutate_rows"](
```

Comment:
Is this already handled in read rows or does it need to be added there as well?

Reply:
all methods that changed: https://github.com/googleapis/googleapis/commit/a175708acd7a367ab3ed120fa562331e1761d825

Reply:
Solid call out. this may need to be done broadly to manage the initial setup calls.

Reply:
both "read_rows" and "sample_rows" return the raw api call right now. So I think this is complete for now, as we do no special handling beyond gapic configs for these.